packages/slate-editor/src/modules/editor-v4-rich-formatting/components/BlockQuoteElement/BlockQuoteElement.tsx

```diff
@@ -19,15 +21,35 @@ export const BlockQuoteElement: FunctionComponent<Props> = ({
     className,
     element,
     ...props
-}) => (
-    <blockquote
-        {...attributes}
-        {...props}
-        className={classNames('editor-v4-block-quote-element', className)}
-        data-slate-type={ElementType.BLOCK_QUOTE}
-        data-slate-value={JSON.stringify(element)}
-        style={{ textAlign: element.align }}
-    >
-        {children}
-    </blockquote>
-);
+}) => {
+    const align = element.align ?? Alignment.LEFT;
+    const isEmpty = Node.string(element).length === 0;
+
+    return (
+        <blockquote
+            {...attributes}
+            {...props}
+            className={classNames('editor-v4-block-quote-element', className, {
+                ['editor-v4-block-quote-element--alignLeft']: align === Alignment.LEFT,
+                ['editor-v4-block-quote-element--alignCenter']: align === Alignment.CENTER,
+                ['editor-v4-block-quote-element--alignRight']: align === Alignment.RIGHT,
+            })}
+            data-slate-type={ElementType.BLOCK_QUOTE}
+            data-slate-value={JSON.stringify(element)}
+        >
+            <p
+                data-placeholder={isEmpty ? 'Quote' : undefined}
```

Comment:
This `isEmpty` check does not take into account void elements: void elements do not have text content, but cannot be considered empty. This causes the prompt to overlap with a publication date placeholder:

![Screenshot-2022-05-09,13-40-31](https://user-images.githubusercontent.com/370680/167394023-f4e29733-cbe6-45d4-8b41-ee46ea6bd69f.png)
 
See how `isEmptyParagraphElement` is implemented.